src/Goerli-DssSpell.sol

```diff
@@ -46,6 +71,25 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         return false;
     }
 
+    address internal immutable DAI = DssExecLib.dai();
+
+    address internal immutable TELEPORT_JOIN = DssExecLib.getChangelogAddress("MCD_JOIN_TELEPORT_FW_A");
+    address internal immutable ROUTER        = DssExecLib.getChangelogAddress("MCD_ROUTER_TELEPORT_FW_A");
+
+    bytes32 internal constant ILK        = "TELEPORT-FW-A";
+    bytes32 internal constant DOMAIN_ETH = "ETH-GOER-A";
```

Comment:
Not used.

Reply:
Removed in [f52b7eb](https://github.com/makerdao/spells-goerli/pull/131/commits/f52b7eb8005d65f0b59e6a9fd2ebbe51a8592aac)